File: recaudacion/2.Codigo/frontend/src/app/core/models/pide.ts
export class Reniec {
    numeroDni: string;
    apellidoPaterno: string;
    apellidoMaterno: string;
    nombres: string;
    paisNacimiento: string;
    departamentoNacimiento: string;
    provinciaNacimiento: string;
    distritoNacimiento: string;
    direccion: string;
    domicilioApp: string;
    nombreCompleto: string;
}

export class Migracion {
    strCalidadMigratoria: string;
    strNombres: string;
    strNumRespuesta: string;
    strPrimerApellido: string;
    strSegundoApellido: string;
    strNombreCompleto: string;
}


export class Sunat {
    cod_dep: string;
    cod_dist: string;
    cod_prov: string;
    ddp_ciiu: string;
    ddp_doble: string;
    ddp_estado: string;
    ddp_fecact: string;
    ddp_fecalt: string;
    ddp_fecbaj: string;
    ddp_flag22: string;
    ddp_identi: string;
    ddp_inter1: string;
    ddp_lllttt: string;
    ddp_mclase: string;
    ddp_nombre: string;
    ddp_nomvia: string;
    ddp_nomzon: string;
    ddp_numer1: string;
    ddp_numreg: string;
    ddp_numruc: string;
    ddp_reacti: string;
    ddp_refer1: string;
    ddp_secuen: string;
    ddp_tamano: string;
    ddp_tipvia: string;
    ddp_tipzon: string;
    ddp_tpoemp: string;
    ddp_ubigeo: string;
    ddp_userna: string;
    desc_ciiu: string;
    desc_dep: string;
    desc_dist: string;
    desc_estado: string;
    desc_flag22: string;
    desc_identi: string;
    desc_numreg: string;
    desc_prov: string;
    desc_tamano: string;
    desc_tipvia: string;
    desc_tipzon: string;
    desc_tpoemp: string;
    esActivo: boolean;
    esHabido: boolean;
    desc_domi_fiscal:string;
    representante: any;
}

export class SunatRepresentante {
    attributes: string;
    cod_cargo: string;
    cod_depar: any;
    desc_docide: string;
    num_ord_suce: string;
    rso_cargoo: string;
    rso_docide: string;
    rso_fecact: string;
    rso_fecnac: string;
    rso_nombre: string;
    rso_nrodoc: string;
    rso_numruc: string;
    rso_userna: any;
    rso_vdesde: string;
}

File: recaudacion/2.Codigo/backend/RecaudacionApiComprobantePago/Domain/Pide.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace RecaudacionApiComprobantePago.Domain
{
    public class Pide
    {
    }

    public class Migracion
    {
        public string strCalidadMigratoria { get; set; }
        public string strNombres { get; set; }
        public string strNumRespuesta { get; set; }
        public string strPrimerApellido { get; set; }
        public string strSegundoApellido { get; set; }
        public string strNombreCompleto { get; set; }
    }

    public class Sunat
    {
        public string cod_dep { get; set; }
        public string cod_dist { get; set; }
        public string cod_prov { get; set; }
        public string ddp_ciiu { get; set; }
        public string ddp_doble { get; set; }
        public string ddp_estado { get; set; }
        public string ddp_fecact { get; set; }
        public string ddp_fecalt { get; set; }
        public string ddp_fecbaj { get; set; }
        public string ddp_flag22 { get; set; }
        public string ddp_identi { get; set; }
        public string ddp_inter1 { get; set; }
        public string ddp_lllttt { get; set; }
        public string ddp_mclase { get; set; }
        public string ddp_nombre { get; set; }
        public string ddp_nomvia { get; set; }
        public string ddp_nomzon { get; set; }
        public string ddp_numer1 { get; set; }
        public string ddp_numreg { get; set; }
        public string ddp_numruc { get; set; }
        public string ddp_reacti { get; set; }
        public string ddp_refer1 { get; set; }
        public string ddp_secuen { get; set; }
        public string ddp_tamano { get; set; }
        public string ddp_tipvia { get; set; }
        public string ddp_tipzon { get; set; }
        public string ddp_tpoemp { get; set; }
        public string ddp_ubigeo { get; set; }
        public string ddp_userna { get; set; }
        public string desc_ciiu { get; set; }
        public string desc_dep { get; set; }
        public string desc_dist { get; set; }
        public string desc_estado { get; set; }
        public string desc_flag22 { get; set; }
        public string desc_identi { get; set; }
        public string desc_numreg { get; set; }
        public string desc_prov { get; set; }
        public string desc_tamano { get; set; }
        public string desc_tipvia { get; set; }
        public string desc_tipzon { get; set; }
        public string desc_tpoemp { get; set; }
        public string desc_domi_fiscal { get; set; }
        public bool esActivo { get; set; }
        public bool esHabido { get; set; }
    }

    public class Reniec
    {
        public string numeroDni { get; set; }
        public string digitoVerificacion { get; set; }
        public string apellidoPaterno { get; set; }
        public string apellidoMaterno { get; set; }
        public string nombres { get; set; }
        public string domicilioApp { get; set; }
        public string nombreCompleto { get; set; }

    }
}
